### PR TITLE
[DOC] Add french train data notebook

### LINKED ISSUE #438: [DOC] French train data notebook
# Brief Description

I'd like to write a notebook that leverage French train data to illustrate some basic pyjanitor functions and the difference between adding new column statistics in the call chain verses one at a time.

(optional but encouraged) This notebook would likely cover the following pyjanitor functions:

- clean_names
- remove_empty
- rename_column
- drop_columns
- join_apply, add_columns

# Dataset
https://raw.githubusercontent.com/rfordatascience/tidytuesday/master/data/2019/2019-02-26/small_trains.csv

French train data source - National Society of French Railways

### PR DESCRIPTION
**Pull request by puruckertom**
# PR Description

Please describe the changes proposed in the pull request: 

Use a French train dataset to illustrate some basic pyjanitor functions and the difference between adding new column statistics in the call chain verses one at a time.

- clean_names
- remove_empty
- rename_column
- drop_columns
- join_apply, add_columns

**This PR resolves #438.**

# PR Checklist

Please ensure that you have done the following:

1. [x] PR in from a fork off your branch. Do not PR from `<your_username>`:master, but rather from `<your_username>`:<branch_name>.

2. [x] If you're not on the contributors list, add yourself to `AUTHORS.rst`.

## Quick Check

To do a very quick check that everything is correct, follow these steps below:

Only added the python notebook to documentation.

Please tag maintainers to review.

- @ericmjl

### CODE REVIEW

**Comment by puruckertom** [comment]
I was unable to run make format on the windows laptop that I brought to Austin, but merged some more recent commits and switched over to my desktop red hat -- now when I run make format from rhel I get "78 files left unchanged."

File: AUTHORS.rst [hunk 1]
```diff
@@ -39,7 +39,6 @@ Contributors
 - `@Eidhagen <https://github.com/Eidhagen>`_ | `contributions <https://github.com/ericmjl/pyjanitor/pulls?utf8=%E2%9C%93&q=is%3Aclosed+mentions%3AEidhagen>`_
 - `@mdini <https://github.com/mdini>`_ | `contributions <https://github.com/ericmjl/pyjanitor/pulls?utf8=%E2%9C%93&q=is%3Aclosed+mentions%3Amdini>`_
 - `@kimt33 <https://github.com/kimt33>`_ | `contributions <https://github.com/ericmjl/pyjanitor/pulls?utf8=%E2%9C%93&q=is%3Aclosed+mentions%3Akimt33>`_
-- @jack-kessler-88 | user no longer found
```
Comment (ericmjl) [inline AUTHORS.rst]: Though @jack-kessler-88 is not found, they should remain on the contributors list, because Jack as a person does exist :smile:.

Comment (ericmjl) [reply AUTHORS.rst]: This may have just been an error when merging. I'll just add it back.

**Comment by ericmjl** [comment]
Thank you for your contribution, @puruckertom!

**Pull request merged**